bundle/bin/rke2-uninstall.sh

```diff
@@ -65,9 +65,9 @@ uninstall_disable_services()
 }
 
 uninstall_remove_files()
-{   
-    
-    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
+{
+
+    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ] || [ -r /etc/rocky-release ] || [ -r /etc/amazon-linux-release ]; then
```

Comment:
Please remove section on rocky release. QA does not have bandwidth to validate changes to rocky installs this release cycle so we want to restrict the scope of this PR.
```suggestion
    if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ] || [ -r /etc/amazon-linux-release ]; then
```